internal/modifiers/timemodifier/replace_if_outdated.go

```diff
@@ -43,24 +42,14 @@ func (r RequiresReplaceModifier) Modify(ctx context.Context, req tfsdk.ModifyAtt
 		return
 	}
 
-	var rotationRFC3339 types.String
+	var rotationRFC3339 rfc3339type.Value
```

Comment:
Could consider `timestamp.RFC3339` for consistency with `types.<Value>` from the Framework.

Reply:
Oooo I like what you're thinking here. That package could be updated to `timetypes` so this becomes `RFC3339` like you suggest. I'm not sure that we should necessarily support anything outside (if we do wind up adopting the types "officially") since RFC3339 is supposed to be the _conventional_ timestamp format across Terraform, but this would leave room for other, similar types, if desired.